SSFWServer/SSFWClass.cs

```diff
@@ -307,10 +321,17 @@ private static HttpResponse SSFWRequestProcess(HttpRequest request, HttpResponse
                                 if (putbuffer != null)
                                 {
                                     Directory.CreateDirectory(directoryPath);
+
+                                    if(!Directory.Exists(savaDataAvatarFilePath))
+                                        Directory.CreateDirectory(savaDataAvatarFilePath);
```

Comment:
No, C# already check this internally, simply create directory.

Reply:
ahh ok i did this just to ensure, good point.